### PR TITLE
xflux: new package

### PR DESCRIPTION
**Pull request by paholg**
I created a package for [xflux](https://justgetflux.com/linux.html).

> [xflux] makes the color of your computer's display adapt to the time of day, warm at night and like sunlight during the day. 

I'm new to this, so please let me know if I did anything wrong.

I could change the url point to [this github repository](https://github.com/Kilian/f.lux-indicator-applet), which is distinct from the creator of the program, but is linked to from their website.

I think I did the licensing correct (as unfortunate as it is), but it's available [here](https://justgetflux.com/news/pages/eula/).

### CODE REVIEW

File: pkgs/tools/misc/xflux/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  libPath = stdenv.lib.makeLibraryPath [
+    stdenv.cc.gcc
+    libXxf86vm
+    libXext
+    libX11
+    libXrandr
+  ];
+
+  phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
```
Comment (lethalman) [inline pkgs/tools/misc/xflux/default.nix]: You don't need this.

File: pkgs/tools/misc/xflux/default.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  unpackPhase = ''
```
Comment (lethalman) [inline pkgs/tools/misc/xflux/default.nix]: You don't need this either.

File: pkgs/tools/misc/xflux/default.nix [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    unpackFile $src;
+  '';
+  installPhase = ''
+    mkdir -p "$out/bin"
+    cp  xflux "$out/bin"
+  '';
+  fixupPhase = ''
```
Comment (lethalman) [inline pkgs/tools/misc/xflux/default.nix]: Use `postFixup`

**Comment by paholg** [comment]
I got rid of the phases line and changed fixupPhase to postFixup. However, if I get rid of the unpackPhase section, then nix throws an error as it expects the tarball to contain a directory and it does not.

I also realize now that I should have created a branch for this -- should I close this and create a new pull request on a separate branch?

**Comment by lethalman** [comment]
It's ok with that `unpackPhase = true`. Yes please cleanup the PR, it's unmergeable like this because github messed up with the commits.

**Pull request closed without merging**